src/Client.php

```diff
@@ -1,8 +1,8 @@
 <?php
 
-namespace pusher;
+namespace Pusher;
 
-use pusher\Version;
+use Pusher\Version;
```

Comment:
Imports don't work like that in php. They have nothing to do with autoloading at all. :)